test/users/npadmana/gsl_demonstration/SKIPIF

```diff
@@ -1,8 +1,25 @@
 #!/usr/bin/env python
 
 #
-# skip this test when FFTW_DIR is not set
+# This test requires:
+#  * CHPL_LLVM!=none (since the test uses extern blocks)
+# and a GSL installation detected like so:
+#  * GSL_DIR environment variable is set, OR
+#  * GSL dynamic library is detected
 #
+# Note that if the dynamic library is found, this
+# test assumes that the header and static library are available.
 
-import os
-print('GSL_DIR' not in os.environ)
+from __future__ import print_function
+from ctypes.util import find_library
+from os import environ
+
+if environ['CHPL_LLVM'] == 'none':
```

Comment:
CHPL_LLVM is guaranteed to be in the environment by our testing system, as far as I know. In particular:
 *  `util/test/testEnv` runs skipif files
 * `testEnv` sets the environment based upon the output of `util/printchplenv --sh`
 * `util/printchplenv` always includes CHPL_LLVM